src/backends/webgl.ts

```diff
@@ -96,7 +137,7 @@ function get_prog_pair(ir: CompilerIR, progid: number) {
 
   // Get the fragment program.
   if (vertex_prog.quote_children.length > 1 ||
-      vertex_prog.quote_children.length < 1) {
+    vertex_prog.quote_children.length < 1) {
```

Comment:
I know this is a tiny thing, but it looks like some whitespace changes might have gotten accidentally committed. Some of them look 100% fine, but this one makes the code a bit harder to read because the condition is no longer aligned with the opening (.